src/analysis/exploration/printLabeledData.m

```diff
@@ -17,7 +17,8 @@ function printLabeledData(labels, data, nonzeroFlag, sortCol, fileName, headerRo
 % .. Authors: Markus Herrgard 6/9/06
 
 tol = 1e-9;
-format = '%g\t';
+%format = '%g\t';
```

Comment:
We need to have a check here then, as this format fails on Windows (the `%g` is cross platform).